components/shell/bash/Makefile

```diff
@@ -31,6 +31,7 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		bash
 COMPONENT_VERSION=	5.2
+COMPONENT_REVISION=	1
 COMPONENT_FMRI=		shell/bash
```

Comment:
You don't need to set COMPONENT_REVISION here. Later the patch level will be automatically considered when IPS_COMPONENT_VERSION is being set.

Reply:
Ah, thanks. Updated